frontend/src/App.js

```diff
@@ -26,6 +27,26 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function App() {
   const classes = useStyles();
+  const matchDataRefreshRate = 5000;
```

Comment:
Our site is kinda slow/laggy after you submit now maybe because of refreshing too many times? We should investigate and maybe refresh less frequently

Reply:
Yes, I am going to further investigate on how it reacts once the multiple page views are added and adjust accordingly. Thank you